account_reconcile_compassion/views/account_invoice_view.xml

```diff
@@ -10,6 +10,7 @@
                 <button name="change_attribution" type="object" string="Reconcile" class="oe_highlight"
                         attrs="{'invisible': [('to_reconcile', '=', 0.0)]}"/>
             </xpath>
+            <xpath expr="//field[@name='team_id']" position="replace"/>
```

Comment:
It's better to make the field invisible instead of removing it, to guarantee compatibility with other modules that might use the field.
```suggestion
            <xpath expr="//field[@name='team_id']" position="attributes">
                <attribute name="invisible">1</attribute>
            </xpath>
```